contracts/optimism_bridge/AnteOptimismL1BridgeTokenSolvencyTest.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import {AnteTest} from "../AnteTest.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+interface L1StandardBridge {
```

Comment:
Tiny code styling suggestion: rename to `IL1StandardBridge` (prefix with capital "i")